po/fr.po

```diff
@@ -521,47 +541,98 @@ msgstr ""
 "GNOME ci-dessous et vérifiez si vos extensions seront toujours disponibles."
 
 #: src/exm-window.blp:40
+msgid "Main Menu"
+msgstr "Menu principal"
+
+#: src/exm-window.blp:53
+#| msgctxt "Navigation"
 msgctxt "Navigation"
```

Comment:
```suggestion
msgctxt "Navigation"
```

CI complains about line 549, but removing `#| msgctxt "Navigation"` (line 548) should fix it.